### ISSUE TITLE
bug: Ios date input centres the value

### ISSUE THREAD

**Issue by milanharia** [ionic-team/ionic-framework#24197]
### Prerequisites

- [x] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [x] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [x] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [ ] v5.x
- [x] v6.x

### Current Behavior

Currently on iOS, the date input centres the value for date which means it does not line up with the rest of the inputs.

https://user-images.githubusercontent.com/76167627/141130988-bc69aeb5-df51-4674-a53c-9b4c480c54b9.mp4

### Expected Behavior

I expect the date to be displayed at the beginning of the input to align with the rest of the fields in the form. See the screenshot (on web) below:

![Expected Behaviour](https://user-images.githubusercontent.com/76167627/141131678-e1cb9f09-349a-414a-8cef-047aba09a35b.png)

### Steps to Reproduce

Create a date input with the following code and open the app on iOS.

`<IonItem>`
`<IonLabel position='stacked'>Date</IonLabel>`
`<IonInput type='date' />`
`</IonItem>`

### Ionic Info

Ionic:

   Ionic CLI       : 6.17.1 (/Users/milan/.nvm/versions/node/v14.17.6/lib/node_modules/@ionic/cli)
   Ionic Framework : @ionic/react 6.0.0-rc.2

Capacitor:

   Capacitor CLI      : 3.2.5
   @capacitor/android : not installed
   @capacitor/core    : 3.2.5
   @capacitor/ios     : 3.2.5

Utility:

   cordova-res : not installed globally
   native-run  : 1.5.0

System:

   NodeJS : v14.17.6 (/Users/milan/.nvm/versions/node/v14.17.6/bin/node)
   npm    : 7.24.2
   OS     : macOS Big Sur

**Comment by milanharia** [comment]
Looking into the dev tools, I managed to find what's breaking it. If you add this CSS it will solve the issue as a temporary fix.

```
input::-webkit-date-and-time-value {
    text-align: left!important;
}
```

**Comment by sean-perkins** [comment]
This has been fixed in `^6.0.0-rc.3`.
